Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -46,10 +47,13 @@ public MainWindow()
             InitializeComponent();
         }
 
-        private void OnClosing(object sender, CancelEventArgs e)
+        private async void OnClosing(object sender, CancelEventArgs e)
```

Comment:
What happens when user use sys command exit, will it route the call here?

Reply:
sys exit will call MainWindow.Close()